### PR TITLE
Feature/json conversion

### PR DESCRIPTION
**Pull request by sschmittBt**

### CODE REVIEW

File: src/test/java/de/braintags/vertx/util/json/TJsonConverter.java [hunk 1]
```diff
@@ -0,0 +1,196 @@
+package de.braintags.vertx.util.json;
```
Comment (remi128) [inline src/test/java/de/braintags/vertx/util/json/TJsonConverter.java:1]: Please add to test suite TestAllUtil

File: src/main/java/de/braintags/vertx/util/json/JsonConverter.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      return convertValueNode(jsonNode);
+    } else if (jsonNode.isArray()) {
+      JsonArray jsonArray = new JsonArray();
+      ArrayNode arrayNode = (ArrayNode) jsonNode;
+      for (JsonNode subNode : arrayNode) {
+        jsonArray.add(convertJsonNodeToVertx(subNode));
+      }
+      return jsonArray;
+    } else {
+      JsonObject result = JsonObject.mapFrom(jsonNode);
```
Comment (remi128) [inline src/main/java/de/braintags/vertx/util/json/JsonConverter.java]: why not return the object directly?

**Review by remi128 — CHANGES_REQUESTED** [review]

File: src/test/java/de/braintags/vertx/util/json/TJsonConverter.java (hunk 1 above)
Comment (sschmittBt) [reply src/test/java/de/braintags/vertx/util/json/TJsonConverter.java:1]: Done

File: src/main/java/de/braintags/vertx/util/json/JsonConverter.java (hunk 2 above)
Comment (sschmittBt) [reply src/main/java/de/braintags/vertx/util/json/JsonConverter.java]: Fixed

**Review by remi128 — APPROVED** [review]

**Pull request merged**